### PR TITLE
Bootstrap3 Migration: Messaging > SMS Connectivity

### PR DESCRIPTION
**Pull request by pnaw94**
@biyeun 

screenshot01
screenshot02

### CODE REVIEW

**Comment by czue** [comment]
hello @pnaw94! welcome to the commcare project team :)

**Comment by gcapalbo** [comment]
thanks @pnaw94 , I will test this out in the coming days and provide feedback. in the meantime, i'm also pinging @biyeun in case she wants to have a look as well

File: corehq/apps/sms/templates/sms/gateway_list.html [hunk 1]
```diff
@@ -10,74 +10,76 @@
...
+                            </a>
+                            <a href="#" data-dismiss="modal" class="btn btn-default">{% trans "Cancel" %}</a>
+                        </div>
+                    </div>
                 </div>
             </div>
             <!-- /ko -->
-            <!-- ko if: isGlobal --><span class="label"><i class="icon-info-sign"></i> {% trans 'Global Connection' %}</span><!-- /ko -->
-            <!-- ko if: isShared --><span class="label"><i class="icon-info-sign"></i> {% trans 'Shared Connection' %}</span><!-- /ko -->
+            <!-- ko if: isGlobal --><span class="label label-info"><i class="fa fa-info-circle"></i> {% trans 'Global Connection' %}</span><!-- /ko -->
```
Comment (biyeun) [inline corehq/apps/sms/templates/sms/gateway_list.html]: i would use `label-default` instead

**Comment by biyeun** [comment]
@gcapalbo looks good to me except that one comment...will leave you to sign off

**Comment by pnaw94** [comment]
Updated

**Pull request merged**